### PR TITLE
Feature/feature flag on mobile app (Gavin)

### PR DESCRIPTION
**Pull request by bernarden**

### CODE REVIEW

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+**3. Adding feature flag to the mobile app**
+
+**4. Validate that feature flag is operational**
+
+
+
+##Tasks
+### Task 1: Clone repository using Git ###
+Let's get the PartUnlimited's source code. If you already have the source code on your machine then skip to the next task.
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: PartsUnlimited

File: src/PartsUnlimited.Mobile/PartsUnlimited/Resources/GlobalResources.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel;
+
+namespace PartsUnlimited.Resources
+{
+    public class GlobalResources : INotifyPropertyChanged
+    {
+        // Singleton
+        public static GlobalResources Current = new GlobalResources();
+
+        public string Website { get; } = "https://www.microsoft.com";
```
Comment (gavinbarron) [inline src/PartsUnlimited.Mobile/PartsUnlimited/Resources/GlobalResources.cs]: This should be http://partsunlimitedweb.azurewebsites.net as per the SoW/requirements for this lab

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+	    get { return _isFeatureFlagEnabled; }
+	    set
+	    {
+	        _isFeatureFlagEnabled = value;
+	        OnPropertyChanged(nameof(IsFeatureFlagEnabled));
+	    }
+	}
+	...
+	```
+	> **Note:** If you are already hosting your PartsUnlimited's website in Azure, then change `https://www.microsoft.com` to that App Service's URL (`http://pudncore.azurewebsites.net` in this example).
```
Comment (gavinbarron) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: The existing value for Website needs to be http://partsunlimitedweb.azurewebsites.net. The wording here is a little confusing perhaps: Change the value of Website to match the URL of your existing deployment of the PartsUnlimited website in Azure, e.g. http://pudncore.azurewebsites.net

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+	![](media/7.png)
+
+* Add the following code:
+	```
+	...
+	public string Website { get; } = "http://pudncore.azurewebsites.net";
+
+	private bool _isFeatureFlagEnabled;
+	public bool IsFeatureFlagEnabled
```
Comment (gavinbarron) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Let's use a more descriptive Property name here. perhaps EnableDiscountFeatureFlag

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md (hunk 1 above)
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
+﻿HOL - Feature Flag for a Mobile Solutions
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+- [Windows SDK and emulator](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive) installed  
+
+- [Adnroid SDK](https://developer.android.com/studio/index.html) and [Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) installed
+
+- [Visual Studio Emulator for Android](https://www.visualstudio.com/vs/msft-android-emulator/ ) installed.
+
+- [Xamarin](https://developer.xamarin.com/guides/cross-platform/windows/visual-studio/) installed and set up for Visual Studio.
+
+	> **Note:** This includes [Mac setup](https://developer.xamarin.com/guides/ios/getting_started/installation/windows/connecting-to-mac/), if you have intensions to run iOS app.
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+
+
+## Tasks Overview: ##
+**1. Clone repository using Git**
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+* Click on "File", "Open" and then "Project/Solution".
+
+	![](media/2.png)
+
+* Navigate to the directory where you cloned the repository, select "PartsUnlimited.Mobile.sln" and click on "Open".
+
+	![](media/3.png)
+
+* If you haven't done so already, you will be prompted to connect to Mac machine.
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Mac connection is necessary if they want to compile and run iOS apps. I have added a note saying that.

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+	> **Note:** To connect Mac to your Visial Studio instance follow these instructions: [Xamarin: Connecting to the Mac](https://developer.xamarin.com/guides/ios/getting_started/installation/windows/connecting-to-mac/)
+
+**Step 3.** You should be able to build and run the solution on iOS, Android and UWP(windows phone 10 or windows 10). Let's run it on Windows Phone 10 emulator.
+
+* Right click on "PartsUnlimited.UWP" project, then on "Set as StartUp Project".
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md (hunk 4 above)
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md (hunk 3 above)
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

File: src/PartsUnlimited.Mobile/PartsUnlimited/Resources/GlobalResources.cs (hunk 2 above)
Comment (bernarden) [inline src/PartsUnlimited.Mobile/PartsUnlimited/Resources/GlobalResources.cs]: Done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 10]
```diff
@@ -174,10 +178,10 @@ In this task you will add a feature flag that increases discount for all product
 
 ![](media/12.png)
 
->**Note:** By clicking on the product, the app will navigate you the website specified in `GlobalResource.cs` file.
+>**Note:** By clicking on the product, the app will navigate you to the website specified in `GlobalResource.cs` file.
 
 
 
-##Congratulations, you have completed this lab.
+## Congratulations, you have completed this lab.
```
Comment (gavinbarron) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Good work, thanks :)

**Comment by bernarden** [comment]
Closing due to a project being finished long time ago.

**Pull request closed without merging**